.coveragerc

```diff
@@ -6,6 +6,7 @@ omit =
     .tox/*
     /usr/*
     setup.py
+    pre_commit_hooks/loaderon_hooks/tests/testing_samples/*
```

Comment:
generally I leave this file unedited -- if I'm going to check in additional python files that don't have coverage -- I'd instead prefer to inline those in the test (this also has the added benefit in that it's easy to see from the test what you're testing!)